sorc/ncep_post.fd/LFMFLD.f

```diff
@@ -74,10 +75,11 @@ SUBROUTINE LFMFLD(RH3310,RH6610,RH3366,PW3310)
       use params_mod, only: d00, d50, pq0, a2, a3, a4, h1, d01, gi
       use ctlblk_mod, only: jsta, jend, modelname, spval, im
       use physcons_post, only: con_rd, con_rv, con_eps, con_epsm1
+      use UPP_PHYSICS
```

Comment:
Would you consider the format? 
`use module_name, only: XXX`
Only specify routines/variables which are used in the this routine.

Reply:
OK